bpf/bpf_host.c

```diff
@@ -1121,6 +1121,11 @@ do_netdev(struct __ctx_buff *ctx, __u16 proto, bool from_host)
 	}
 
 	switch (proto) {
+# if defined ENABLE_ARP_PASSTHROUGH || defined ENABLE_ARP_RESPONDER
+	case bpf_htons(ETH_P_ARP):
+		ret = CTX_ACT_OK;
+		break;
+# endif
```

Comment:
Usually we nest this inside `ENABLE_IPV4` so ARP is only handled if IPv4 is enabled overall.

Reply:
Ah, yeah, makes sense. I found a single case (on the 4 existing ARP cases we have) where we do this though, in `bpf_lxc.c`. Since tests are passing here, I'd rather fix the ARP cases in a separate PR. It also probably doesn't need to be backported contrary to this PR.